src/test/java/org/opensearch/security/dlic/rest/api/AccountApiTest.java

```diff
@@ -29,6 +29,7 @@
 public class AccountApiTest extends AbstractRestApiUnitTest {
 
     public static final String ENDPOINT = "/_opendistro/_security/api/account";
+    public static final String ENDPOINT_PLUGINS = "/_plugins/_security/api/account";
 
     @Test
```

Comment:
Use parametrized testing

Reply:
Your are right, but if it is possible let's do it as another improvement, since we have to merge this PR by tomorrow(Monday)

Reply:
Copy and paste approach is not maintainable. It is a must for this PR.

Reply:
Parametrized testing added to all test files.